composer.json

```diff
@@ -50,7 +50,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.4-dev"
+            "dev-symfony-process-4": "1.5-dev"
```

Comment:
that should not be changed at all. You don't need to alias your feature branch globally (and it makes your PR unmergeable btw)